schema/crdb/dbinit.sql

```diff
@@ -836,6 +836,31 @@ CREATE TABLE IF NOT EXISTS omicron.public.silo_quotas (
     storage_bytes INT8 NOT NULL
 );
 
+/**
+ * A view of the amount of provisioned and allocated (set by quotas) resources
+ * on a given silo.
+ */
+CREATE VIEW IF NOT EXISTS omicron.public.silo_utilization 
+AS SELECT
+    c.id AS silo_id,
+    s.name AS silo_name,
+    c.cpus_provisioned AS cpus_provisioned,
+    c.ram_provisioned AS memory_provisioned,
+    c.virtual_disk_bytes_provisioned AS storage_provisioned,
+    q.cpus AS cpus_allocated,
+    q.memory_bytes AS memory_allocated,
+    q.storage_bytes AS storage_allocated
+FROM
+    omicron.public.virtual_provisioning_collection AS c
+    RIGHT JOIN omicron.public.silo_quotas AS q 
```

Comment:
Is this a right join because a silo which has never had a provisioning even would have no entry in `virtual_provisioning_collection`?

Reply:
A provisioning collection can be the fleet, a silo, or a project. I want all `silo_quotas` and whatever entries in the `virtual_provisioning_collection` as match.